src/main/java/seedu/ccacommander/logic/commands/CreateMemberCommand.java

```diff
@@ -1,27 +1,27 @@
-package seedu.address.logic.commands;
+package seedu.ccacommander.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_GENDER;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_GENDER;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.ccacommander.logic.parser.CliSyntax.PREFIX_TAG;
 
-import seedu.address.commons.util.ToStringBuilder;
-import seedu.address.logic.Messages;
-import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.Model;
-import seedu.address.model.member.Member;
+import seedu.ccacommander.commons.util.ToStringBuilder;
+import seedu.ccacommander.logic.Messages;
+import seedu.ccacommander.logic.commands.exceptions.CommandException;
+import seedu.ccacommander.model.Model;
+import seedu.ccacommander.model.member.Member;
 
 /**
- * Adds a member to the address book.
+ * Adds a member to CcaCommander.
  */
 public class CreateMemberCommand extends Command {
 
     public static final String COMMAND_WORD = "createMember";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a member to CCACommander. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a member to CcaCommander. "
```

Comment:
Should this stay as CCACommander? In ExitCommand.java, it is CCACommander. Since our app name is CCACommander, should we should stick to our display name as CCACommander and only use CcaCommander while developing?

Reply:
I saw that there are many files that use CcaCommander, we can just change the ExitCommand to use CcaCommander too then

Reply:
We will use CcaCommander for developer facing, and CCACommander for user facing